db-shared/src/main/java/com/splicemachine/db/shared/common/reference/SQLState.java

```diff
@@ -1044,6 +1044,8 @@ The message identifier (ie. the value that matches a field in this class)
 	String LANG_INVALID_USE_OF_DEFAULT                                 = "42Y85";
 	String LANG_INVALID_SESSION_PROPERTY                               = "42Y86";
 	String LANG_INVALID_SESSION_PROPERTY_VALUE                         = "42Y87";
+	String LANG_INVALID_FORCED_NO_SUBQUERY_FLATTEN_VALUE               = "42Y88";
+	String LANG_INVALID_SUBQUYERY_PROPERTY                             = "42Y89";
```

Comment:
typo: LANG_INVALID_SUBQU{Y,}ERY_PROPERTY

Reply:
Thanks @arnaud-splice I've fixed the typo.